#
# 这个参数的修改是为了便于实现nandboot和SDboot的切换
#
#     当sdboot的BL1阶段要从MOVI_BL2_POS（24k + 1k（sd卡的第0扇区））这个偏移位置开始向DRAM cp代码。
#     虽然启动时只会从nand或SD卡一次性读取16K内容，但是为了统一两种启动方式tiny210-spl.bin的大小，
#     生成24k大小的tiny210-spl.bin(这并不会影响校验和，因为文件尾是由0填充)。
#
Index: board/samsung/mini210/tools/mkv210_image.c
===================================================================
--- a/board/samsung/mini210/tools/mkv210_image.c
+++ b/board/samsung/mini210/tools/mkv210_image.c
@@ -2,8 +2,8 @@
 #include <string.h>
 #include <stdlib.h>
 
-#define BUFSIZE                 (8*1024)
-#define IMG_SIZE                (8*1024)
+#define BUFSIZE                 (24*1024)
+#define IMG_SIZE                (24*1024)
 #define SPL_HEADER_SIZE         16
 //#define FILE_PERM               (S_IRUSR | S_IWUSR | S_IRGRP \
                                 | S_IWGRP | S_IROTH | S_IWOTH)
Index: board/samsung/tiny210/tools/mkv210_image.c
===================================================================
--- a/board/samsung/tiny210/tools/mkv210_image.c
+++ b/board/samsung/tiny210/tools/mkv210_image.c
@@ -2,8 +2,8 @@
 #include <string.h>
 #include <stdlib.h>
 
-#define BUFSIZE                 (8*1024)
-#define IMG_SIZE                (8*1024)
+#define BUFSIZE                 (24*1024)
+#define IMG_SIZE                (24*1024)
 #define SPL_HEADER_SIZE         16
 //#define FILE_PERM               (S_IRUSR | S_IWUSR | S_IRGRP \
                                 | S_IWGRP | S_IROTH | S_IWOTH)
--- a/spl/Makefile	2012-12-02 14:01:21.000000000 +0800
+++ b/spl/Makefile	2013-01-13 19:49:17.719100547 +0800
@@ -103,6 +103,7 @@
 $(obj)$(BOARD)-spl.bin: $(obj)u-boot-spl.bin
 	$(TOPDIR)/board/$(BOARDDIR)/tools/mk$(BOARD)spl.exe \
 		$(obj)u-boot-spl.bin $(obj)$(BOARD)-spl.bin
+	cat $(obj)$(BOARD)-spl.bin $(TOPDIR)/u-boot.bin > $(TOPDIR)/u-boot-$(BOARD).bin
 endif
 
 $(obj)u-boot-spl.bin:	$(obj)u-boot-spl
